microsetta_interface/routes.yaml

```diff
@@ -385,6 +385,22 @@ paths:
               schema:
                 type: string
 
+  # same as above
+  '/accounts/{account_id}/request/remove':
+    post:
```

Comment:
confirm

Reply:
confirmed this is the endpoint called by the 'Delete Account' button in the Accounts Details page.